src/resources/js/v2/tickets-loader.js

```diff
@@ -0,0 +1,93 @@
+/* global tribe */
+/**
+ * Makes sure we have all the required levels on the Tribe Object
+ *
+ * @since TBD
+ *
+ * @type   {PlainObject}
+ */
+tribe.tickets = tribe.tickets || {};
+
+/**
+ * Configures ET Loader Object in the Global Tribe variable
+ *
+ * @since TBD
+ *
+ * @type   {PlainObject}
+ */
+tribe.tickets.loader = {};
```

Comment:
from a bigger perspective, I wonder if having a common JS for the loader component is a good idea.

Reply:
Yes, that would be an option. I feel it could become tricky when some plugin changes the loader templates (each plugin has their own), depending on their needs. This could be abstracted in common if we make sure the main template structure doesn't change.